ui-tests-protractor/e2e/page-objects/connections/connection.po.ts

```diff
@@ -0,0 +1,38 @@
+import { element, by, ElementFinder, ElementArrayFinder } from 'protractor';
+import { P } from '../../common/world';
+import { ConnectionsListComponent } from './list/list.po';
+import {log} from '../../../src/app/logging';
+
+/**
+ * Created by mcada on 7.11.17.
+ */
+export class ConnectionObjectElement {
```

Comment:
I still have a feeling that this class duplicates ConnectionList, or at least it could live in the same file.